sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_get_bounding_boxes_async.py

```diff
@@ -52,7 +50,7 @@ async def get_bounding_boxes(self):
             # Make sure your form's type is included in the list of form types the custom model can recognize
             with open(path_to_sample_forms, "rb") as f:
                 forms = await form_recognizer_client.recognize_custom_forms(
```

Comment:
can fix this later but just a heads up - looks like we might get an unclosed client session warning here

Reply:
the `async with` is there, just hidden in the collapsed lines